dashboard/config/scripts/levels/iceage_repeat.level

```diff
@@ -1,7 +1,7 @@
 <Studio>
   <config><![CDATA[{
   "game_id": 33,
-  "created_at": "2018-12-14T19:32:14.000Z",
+  "created_at": "2015-11-30T21:36:41.000Z",
```

Comment:
when you have time, it's a good idea to occasionally turn off `levelbuilder_mode` and run `rake seed:all`. This will overwrite many differences between what's in the filesystem (including these .level files) and what's in your database, and should make it so that the next time you run `rake seed:all` or `rake seed:scripts` you won't see any local differences show up.